changelog

```diff
@@ -3,12 +3,14 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 vNext
 ----------
 -[MINOR] Support for email OTP MFA in native authentication (#2167)
+-[MINOR] Add legacy FIDO2 API version (#2188)
 
 Version 5.6.0
 ----------
-- [PATCH] Update common @18.0.0
+-[PATCH] Update common @18.0.0
 -[MINOR] Moving OS version check for passkeys (#2138)
 -[PATCH] Fix Native Auth authority data being persisted across different SDK instances (#2159)
+-[MINOR] Support for email OTP MFA in native authentication (#2167)
```

Comment:
Is this meant to be here?

Reply:
No, fixed: a7904b8ef236aef729b8483da236795695667bb6